modules/yarn.lock

```diff
@@ -12856,6 +12856,16 @@ react-text-mask@5.4.3:
   dependencies:
     prop-types "^15.5.6"
 
+react-transition-group@4.4.1, react-transition-group@^4.4.1:
+  version "4.4.1"
+  resolved "https://registry.yarnpkg.com/react-transition-group/-/react-transition-group-4.4.1.tgz#63868f9325a38ea5ee9535d828327f85773345c9"
+  integrity sha512-Djqr7OQ2aPUiYurhPalTrVy9ddmFCCzwhqQmtN+J3+3DzLO209Fdr70QrN8Z3DsglWql6iY1lDWAfpFiBtuKGw==
+  dependencies:
+    "@babel/runtime" "^7.5.5"
+    dom-helpers "^5.0.1"
+    loose-envify "^1.4.0"
+    prop-types "^15.6.2"
+
```

Comment:
Hmm, this change seems odd. You might try re-generating the yarn.lock by discarding the chages and then re-running `yarn install`